Observer/AddToCart.php

```diff
@@ -8,80 +8,21 @@
 class AddToCart implements ObserverInterface
 {
 
-    /**
-     * @param \Metrilo\Analytics\Helper\Data           $helper
-     * @param \Magento\Catalog\Model\ProductRepository $productRepository
-     * @param \Magento\Framework\App\RequestInterface  $request
-     */
     public function __construct(
-        \Metrilo\Analytics\Helper\Data $helper,
-        \Magento\Catalog\Model\ProductRepository $productRepository,
-        \Magento\Framework\App\RequestInterface $request
+        \Metrilo\Analytics\Helper\Data $helper
```

Comment:
here you will include the new `Models\Events\AddToCart` helper that knows how to generate the JS call for addtocart events